src/Components/GetStartedButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native'
+
+const GetStartedButton = () => {
+    return (
+      <TouchableOpacity
+        style={{
```

Comment:
This style does not really match our prototype. Here is what I see on my test device:

<img src="https://user-images.githubusercontent.com/72236249/224511160-22a508d0-cb89-4102-aa23-31448dfd6786.png" width="200">

The button should be a lot wider. The "Inspect" tab on our Figma prototype actually has some style properties that you can copy and paste to make things easier. You can also copy and paste the style properties for the button text.

Reply:
Jordan if you need help making these changes let me know!

Reply:
Requested Changes:
- In the Button Style (After line 7), you can add a height & width property to the Style object.
- In the Text Style (Line 16), change the font size to be the same as what was shown in our Figma prototype.

How I would do it:
- I would first have our Figma prototype pulled up for reference along with our Vs code open: [https://www.figma.com/file/uZyWHhMD4wBsT5ZOnoBmhS/App-Prototype?node-id=2%3A12&t=Q9BekUFKtPoLxvzR-0](url) 
- To view each component of the start button for example, I go to the hierarchy and click on the desired component. Then I look to the tab on the right side of my screen to view the specs such as the sizes of its Width, Height, Font, Border Radius etc.
<img width="1440" alt="Screenshot 2023-03-15 at 9 04 25 AM" src="https://user-images.githubusercontent.com/114822147/225321826-e35a4776-81bc-4325-80d0-816deef7b280.png">